### ISSUE TITLE
addButtonFromModelFunction with a DropDown buttons

### ISSUE THREAD

**Issue by helesjunior** [Laravel-Backpack/CRUD#1737]
# Bug report
I'm having trouble adding an additional button to table actions.

### What I did:
In controller:
`$this->crud->addButtonFromModelFunction('line', 'more', 'moreOptions', 'beginning');`

In Model:
```
public function moreOptions($crud = false)
    {
        $button = '<div class="btn-group">
	                    <button type="button" title="Mais" class="btn btn-xs btn-default dropdown-toggle"
                                            data-toggle="dropdown"><i class="fa fa-gear"></i> Mais
                                        <span class="fa fa-caret-down"></span></button>
                                    <ul class="dropdown-menu" role="menu">
                                         <li><a href="#">Teste 1</a></li>
                                          <li><a href="#">Teste 2</a></li>
                                    </ul>
                        </div>';

        return $button;

    }
```

### What happened:
The menu is not on top the table.
![image](https://user-images.githubusercontent.com/34820135/49302189-86082380-f4ad-11e8-9af7-8c06b3e0a320.png)

### Backpack, Laravel, PHP, DB version:
BackPack 3.5 With Laravel 5.7

**Comment by pxpm** [comment]
Dunno if it will work, i didn't test, but have you tried to add to css .dropdown-menu: { z-index:99999; } ??

**Comment by helesjunior** [comment]
Unfortunately it did not work either.....   :(

**Comment by helesjunior** [comment]
Excellent @johnpuddephatt, commenting on line 66 of the archive **public/crud/css/list.css**, solved the problem!

![image](https://user-images.githubusercontent.com/34820135/49451704-ed341980-f7c6-11e8-8173-ea48c0824b55.png)

**Comment by johnpuddephatt** [comment]
@pxpm i don't think that will work, it'll position the dropdown relatively instead of absolutely meaning it won't 'act' as a dropdown. With overflow: hidden set the only child element that can overflow the parent is a position fixed element, and that's not much use.
